tools/cli/lib/api.c

```diff
@@ -671,6 +684,272 @@ TDNFCliRepoSyncCommand(
     goto cleanup;
 }
 
+uint32_t parse_depResult(char **ppszDependencies, char **ppszVal)
```

Comment:
You can use `TDNFJoinArrayToString()` from `common/strings.c` and make this function much simpler.

Also, use another name for it - it's not parsing anything. It also has nothing special about dependencies, it's just sorting an array of strings and joining them. Maybe `TDNFJoinArrayToStringSorted()`, and move it to `common/utils.c`.

Reply:
Done